FreshRSS/FreshRSS
# fix: sharing service for mailto links for web mail services

Closes #2666

Firefox does not open the mailto link in a new tab, when a web mailer is set in the browser settings.
https://bugzilla.mozilla.org/show_bug.cgi?id=646552

Add a 2nd email sharing service:

![grafik](https://user-images.githubusercontent.com/1645099/193577971-44849efd-3250-460b-a7fe-bcbc9dee7466.png)


Changes proposed in this pull request:

- new sharing service
- Javascript: use window.open() for this service

How to test the feature manually:
1. Firefox: set the mailto handler to f.e. Gmail (see: https://support.mozilla.org/en-US/kb/change-program-used-open-email-links)

1. go to configure -> sharing
2. add the new service
3. go to normal view
4. share an article
5. Gmail will opened

Pull request checklist:
